src/Host/Startup.cs

```diff
@@ -33,7 +35,11 @@ public void ConfigureServices(IServiceCollection services)
             // use in memory for testing.
             services
                 .AddDbContext<BloggingContext>(opt => opt.UseInMemoryDatabase())
-                .AddUnitOfWork<BloggingContext>();
+                .AddUnitOfWork<BloggingContext>()
```

Comment:
We cannot assume the custom repository only dependency DbContext, that's to say, we should get the custom repository from DI.